packages/flutter_tools/lib/src/build_runner/build_runner.dart

```diff
@@ -61,12 +62,13 @@ class BuildRunner {
   /// The defines of the build command are the arguments required in the
   /// flutter_build kernel builder.
   Future<BuildResult> build({
```

Comment:
It seems this method has side effects in addition to returning a `BuildResult`, e.g. when `disableKernelGeneration` is set to `false`. Can you can that to the dartdocs?